tests/test_create_incident.py

```diff
@@ -35,40 +55,31 @@ def test_admin_cannot_create_a_red_flag(client):
 
 
 def test_create_a_red_flag_without_data(client):
-    response = client.post("api/v2/red-flags", headers=user1_header)
+    response = client.post("api/v2/incidents", headers=user1_header)
     assert response.status_code == 400
     data = json.loads(response.data.decode())
-    assert data["error"] == "Please provide red-flag Data"
+    assert data["error"] == "Please provide incident Data"
 
 
 def test_create_a_red_flag_with_valid_data(client):
     new_red_flag[
         "comment"
     ] = "Lorem ipsum eiusmod temport labore et dolore magna"
     response = client.post(
-        "api/v2/red-flags", headers=user1_header, data=json.dumps(new_red_flag)
+        "api/v2/incidents", headers=user1_header, data=json.dumps(new_red_flag)
     )
     assert response.status_code == 201
     data = json.loads(response.data.decode())
-    assert data["data"][0]["message"] == "Created red-flag record"
+    assert data["data"][0]["success"] == "Created red-flag record"
     assert data["data"][0]["red-flag"]["created_by"] == user1_id
 
 
-def test_create_a_duplicate_red_flag(client):
-    response = client.post(
-        "api/v2/red-flags", headers=user1_header, data=json.dumps(new_red_flag)
-    )
-    assert response.status_code == 409
-    data = json.loads(response.data.decode())
-    assert data["error"] == "red-flag record already exists"
-
-
 def test_create_a_red_flag_without_wrong_input(client):
```

Comment:
correct function name to match the functionality

Reply:
Thanks, @ManuelDominic. Let me make the correction.